cmd/gapit/trace.go

```diff
@@ -195,13 +195,16 @@ func (verb *traceVerb) captureADB(ctx context.Context, flags flag.FlagSet, start
 	)
 
 	if options.monitorLogcat {
+		d.Command("logcat", "--clear").Run(ctx)
```

Comment:
This is what the `-T 0` is supposed to do.

Reply:
ah, I must have been confused then, now removed.